corehq/apps/reports/standard/cases/case_list_explorer.py

```diff
@@ -176,8 +182,11 @@ def headers(self):
     @property
     def rows(self):
         self.track_search()
-        data = (wrap_case_search_hit(row) for row in self.es_results['hits'].get('hits', []))
-        return self._get_rows(data)
+
+        with self.profiler.timing_context("Retrieving rows"):
+            data = (wrap_case_search_hit(row) for row in self.es_results['hits'].get('hits', []))
```

Comment:
you could consider tracking "Fetching results from ES" and "Wrapping result" as two different metrics, just to be very specific about how long ES took.